packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/nodes/node/dashboard/dashboard.module.js

```diff
@@ -64,6 +68,11 @@ angular
     serverDashboard,
   ])
   .config(routing)
+  .constant('COMMIT_IMPRESSION_TRACKING_DATA', COMMIT_IMPRESSION_TRACKING_DATA)
+  .constant(
+    'RECOMMIT_IMPRESSION_TRACKING_DATA',
+    RECOMMIT_IMPRESSION_TRACKING_DATA,
```

Comment:
You should create an object with all your constants inside. It will avoid referencing a new constant each time it's needed